### ISSUE TITLE
Rticles integration?

### ISSUE THREAD

**Issue by gtalckmin** [davidgohel/officedown#26]
Hi,
Great package. I was wondering if there is some sort of integration with Rticles or a quick way to use docx template for scientific journals?

**Comment by davidgohel** [comment]
Hi,

No, I'm sorry, I don't know much about it. I had looked a little at what rticles does, it should be possible but I have no available time to do it. (If you want to contribute on this, I can help by giving insight on how to extend offiicedown)

**Comment by alapo** [comment]
Saw this post and I got interested. I would be willing to start contributing templates as I submit to articles. I think one way I could help is by showing how to make a template. But I will be honest some of the editing is above my head. I uploaded a YouTube video below with my attempt. @davidgohel  if you're willing to comment. I can remake a better video so people can see how to make their own docx template. There really isnt much out there showcasing the best way to do this.

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/X_U_8Ewa5U0/0.jpg)](https://www.youtube.com/watch?v=X_U_8Ewa5U0)

[MyProject.zip](https://github.com/davidgohel/officedown/files/5283988/MyProject.zip)

The code I used for the RMarkdown is [here](https://pastebin.com/tRrAn5A9) and I also uploaded the project directory.
